### DISCUSSION TITLE
Checkpoint merger does not work

### DISCUSSION THREAD

**Discussion by LeYi100PCT** [acheong08/Diffusion-ColabUI#35] [Q&A] [answered]
It stops working after loading the model, no merging process occurs and no new model is seen.

**Comment by acheong08** [comment]
This seems to be an issue with [stable-diffusion-webui](https://github.com/acheong08/stable-diffusion-webui).
I'll look into it

**Reply by acheong08** [reply]
Have you checked the folders from Google Colab? It might be saving it somewhere but not automatically loaded.

**Reply by LeYi100PCT** [reply]
> Have you checked the folders from Google Colab? It might be saving it somewhere but not automatically loaded.

I don't see it in /models/stable-diffusion/

**Reply by acheong08** [reply]
It should be in /stable-diffusion-webui/checkpoints

**Reply by LeYi100PCT** [reply]
> It should be in /stable-diffusion-webui/checkpoints

Is the folder not there because it was not successfully created?
![YUOJ[G}]]`%A)C(EKK3HPNQ.png](https://user-images.githubusercontent.com/117501683/201525769-2f205393-9f75-4161-bfe0-31a8dbddf217.png)

**Reply by LeYi100PCT** [reply]
What does ^C mean?
![26~%%%R0S$WOY(3W8%C}H(B.png](https://user-images.githubusercontent.com/117501683/201525803-90aecba4-93ce-45ec-9f23-ac8e689afbe1.png)

**Reply by acheong08** [reply]
It means it crashed. I'm not sure why. I'm tracking the issue here: https://github.com/acheong08/Diffusion-ColabUI/issues/36

**Reply by acheong08** [reply]
I'm currently extremely busy with school work so it might take a few days before I can fix it. You might have some luck opening an issue in https://github.com/AUTOMATIC1111/stable-diffusion-webui.

**Comment by acheong08** [comment]
@LeYi100PCT You must set VRAM as `--lowvram`. The crash is caused by the Colab running out of memory

**Reply by LeYi100PCT** [reply]
> @LeYi100PCT You must set VRAM as `--lowvram`. The crash is caused by the Colab running out of memory

It still doesn't work, but the following answer solves the problem:
[https://github.com/AUTOMATIC1111/stable-diffusion-webui/discussions/4537#discussioncomment-4156314](url)

**Reply by acheong08** [reply]
I have no idea why it works but thank you. Should I put this in the Colab as an option?

**Reply by LeYi100PCT** [reply]
Sounds great!

**Comment by acheong08** [comment, marked as answer]
I have fixed the issue by adding a large_models option that automatically runs the answer from [here](https://github.com/AUTOMATIC1111/stable-diffusion-webui/discussions/4537#discussioncomment-4156314)

I am testing it now

**Reply by acheong08** [reply]
@LeYi100PCT Thank you for the solution. It now works!
![image](https://user-images.githubusercontent.com/36258159/202334261-6ad2fcef-6aac-4d4c-90da-96fa7a0dd29e.png)

**Reply by LeYi100PCT** [reply]
Great!
